packages/shared/src/features/onboarding/shared/FunnelStepCtaWrapper.tsx

```diff
@@ -32,7 +32,7 @@ export function FunnelStepCtaWrapper({
   return (
     <div className="relative flex flex-1 flex-col gap-4">
       <div className={classNames('flex-1', containerClassName)}>{children}</div>
-      <div className="sticky bottom-2 m-4 flex flex-col gap-4">
+      <div className="sticky bottom-2 mx-auto my-4 w-full max-w-md px-4 flex flex-col gap-4">
```

Comment:
Full-width steps looked awkward without a max-width on the CTA. 

Now it looks like this:

<img width="1269" alt="image" src="https://github.com/user-attachments/assets/262691d6-f65e-4efe-a3fd-d84509147574" />